content/en/docs/concepts/workloads/pods/disruptions.md

```diff
@@ -116,7 +116,7 @@ is reached.
 
 A PDB specifies the number of replicas that an application can tolerate having, relative to how
 many it is intended to have.  For example, a Deployment which has a `.spec.replicas: 5` is
-supposed to have 5 pods at any given time.  If its PDB allows for there to be 4 at a time,
+supposed to have 5 pods at any given time.  If its PDB allows for there to be 6 at a time,
```

Comment:
Disruptions typically _reduce_ the number of healthy replicas, so I think the original number is right.

Reply:
I just thought line followed by it, which is  `then the Eviction API will allow voluntary disruption of one (but not two) pods at a time"` doesnt match then, cause if number is 4 and it says it distrupts one so number become 3, but as we are working with 5 pods, we want it to be 6 so again if one distribution happens it becomes 5 & number matches to then line: `supposed to have 5 pods at any given time. `

Reply:
Are there more questions or should an issue be raised to clarify the content?

Reply:
This what all i had from my side from & my understanding @kbhawkey